pytorch_pfn_extras/config.py

```diff
@@ -138,6 +138,11 @@ def cache(value: Any) -> Any:
             else:
                 return cache(config)
 
+    def update_via_args(self, args: List[Tuple[str, Any]]) -> None:
```

Comment:
How about using `typing.Sequence` instead?
```suggestion
    def update_via_args(self, args: Sequence[Tuple[str, Any]]) -> None:
```